pkg/proxy/proxy.go

```diff
@@ -61,7 +62,21 @@ func (p *Proxy) isIgnoredUser(committer string) bool {
 		}
 	}
 
-	if committer == "" && p.provider == providers.GithubName {
+	if (committer == "" && p.provider == providers.GithubName){
+		return true
+	}
+
+	return false
+}
+
+func (p *Proxy) isAllowedUser(committer string) bool {
+	if len(p.allowedUsers) > 0 {
+		if exists, _ := utils.InArray(p.allowedUsers, committer); exists {
+			return true
+		}
+	}
+
+	if (committer == "" && p.provider == providers.GithubName){
```

Comment:
@soapy1 This check should be removed, as if committer is empty, we would not want it to be considered as an allowedUser